### ISSUE TITLE
Review info text about original/derivative sentences

### ISSUE THREAD

**Issue by trang** [Tatoeba/tatoeba2#1643]
This information currently is displayed in the "Logs" section of the sentence page, which is maybe not the best place.  As mentioned in https://github.com/Tatoeba/tatoeba2/pull/1623#issuecomment-408595821: 

> The other options I can think of:
>
> 1. Somewhere still in the sidebar but in a separate block. We already have many blocks in the sidebar so I'm not too fond of adding yet another one.
> 2. Next to the sentence id (above the sentence block). We would have to find shorter wordings, or use icons with tooltips.
> 3. Below the sentence id and above the sentence block. Maybe not an optimal use of vertical space. The information is not *that* important to be above the main subject of the page.
> 4. Below the sentence block. I'm afraid some people might think that the information refers to the last translation in the sentence block. Maybe not a big deal.
> 5. In the sentence block itself. It's a quite difficult one, but there's surely a way to fit the information in there. We could then mark the "original" sentence even when we're not on the page of that sentence.

The info text we currently display (there are 3 cases):
- `This sentence is original and was not derived from translation.`
- `This sentence was initially added as a translation of sentence #{n}.`
- `We cannot determine yet whether this sentence was initially derived from translation or not.`

These strings need to be reviewed too.

**Comment by alanfgh** [comment]
My initial suggestions:

- This sentence was not added as a translation.
- This sentence was added as a translation of sentence #{n}.  [that is, use the text you already suggested]
- This sentence may have been added as a translation.

Is there a reason why we need to display the third sentence? Would blank text be an option? Also, why do you suggest "yet"? Is it that we need to perform further analysis before we can make the determination? If so, is it that (a) we don't know how to do the analysis yet but expect to be able to do it in the future, or (b) we don't have the data yet but expect to have it in the future, or (c) we haven't had enough time to run through all the sentences yet or (d) some other possibility?

**Comment by jiru** [comment]
> Is there a reason why we need to display the third sentence? Would blank text be an option?

No particular reason. Yes, I think we could as well have no text at all, considering this information may not be very relevant for most visitors.

> Also, why do you suggest "yet"?

The "yet" means that we need to perform further analysis before we can make 
the determination (reason a). However, while this is true for most of the sentences, there are a few sentences (about 14000) which lack a creation log, so I’m afraid we will actually never be able to find out their origin.
